scripts.js

```diff
@@ -0,0 +1,98 @@
+window.onload = () => {
+  function isValidData(data, pattern) {
+  return pattern.test(data);
+}
+
+
+document.querySelector('#submitButton').disabled = true;
+
+const changeButtonState = function() {
+  if ( 
+    ((isNameEntered && isNumberEntered) || (isCheckboxChecked && isNameEntered)
+    && isRegionSelected 
+    && isCitySeleted)) {
+    document.querySelector('#submitButton').disabled = true;
+  } else {
+    document.querySelector('#submitButton').disabled = false;
+  }
+}
+
+let isNameEntered = false;
+let isNumberEntered = false;
+let isCheckboxChecked = false;
+let isRegionSelected = false;
+let isCitySeleted = false;
+
+document.querySelector('#fullName').addEventListener('change', function(event) {
+  const fullName = document.forms['mainForm'].elements['fullName'].value.trim();
```

Comment:
This code doesn`t work, use names
![image](https://user-images.githubusercontent.com/24321188/58370467-8b4fa380-7f0f-11e9-80c3-bbca3ee9a37f.png)